config/config_test.go

```diff
@@ -2,25 +2,246 @@ package config
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 )
 
+func Test_Getters_Existing_and_Default(t *testing.T) {
+	tc := New()
+	tc.Set("string", "string")
+	require.Equal(t, "string", tc.GetString("string", "default"), "it should return the key value")
+	require.Equal(t, "default", tc.GetString("other", "default"), "it should return the default value")
+
+	tc.Set("bool", false)
+	require.Equal(t, false, tc.GetBool("bool", true), "it should return the key value")
+	require.Equal(t, true, tc.GetBool("other", true), "it should return the default value")
+
+	tc.Set("int", 0)
+	require.Equal(t, 0, tc.GetInt("int", 1), "it should return the key value")
+	require.Equal(t, 1, tc.GetInt("other", 1), "it should return the default value")
+	require.EqualValues(t, 0, tc.GetInt64("int", 1), "it should return the key value")
+	require.EqualValues(t, 1, tc.GetInt64("other", 1), "it should return the default value")
+
+	tc.Set("float", 0.0)
+	require.EqualValues(t, 0, tc.GetFloat64("float", 1), "it should return the key value")
+	require.EqualValues(t, 1, tc.GetFloat64("other", 1), "it should return the default value")
+
+	tc.Set("stringslice", []string{"string", "string"})
+	require.Equal(t, []string{"string", "string"}, tc.GetStringSlice("stringslice", []string{"default"}), "it should return the key value")
+	require.Equal(t, []string{"default"}, tc.GetStringSlice("other", []string{"default"}), "it should return the default value")
+
+	tc.Set("duration", "2ms")
+	require.Equal(t, 2*time.Millisecond, tc.GetDuration("duration", 1, time.Second), "it should return the key value")
+	require.Equal(t, time.Second, tc.GetDuration("other", 1, time.Second), "it should return the default value")
+
+	tc.Set("duration", "2")
+	require.Equal(t, 2*time.Second, tc.GetDuration("duration", 1, time.Second), "it should return the key value")
+	require.Equal(t, time.Second, tc.GetDuration("other", 1, time.Second), "it should return the default value")
+}
+
+func Test_MustGet(t *testing.T) {
+	tc := New()
+	tc.Set("string", "string")
+	require.Equal(t, "string", tc.MustGetString("string"), "it should return the key value")
+	func() {
+		defer func() {
+			if r := recover(); r == nil {
+				t.Errorf("The code did not panic")
+			}
+		}()
+		tc.MustGetString("other")
+	}()
```

Comment:
I think we can replace these blocks with the recover like this:

```suggestion
	require.Panics(t, func() { tc.MustGetString("other") })
```